src/main/scala/exercises/06_Collections.scala

```diff
@@ -31,14 +31,14 @@ object Collections {
    * 
    */
   def firstElementInList[T](l: List[T]): T = {
-    error("fix me")
+    l.head
   }
 
   /**
    * Get the sum of all the elements in the list, e.g. sumOfList(List(1,2,3)) = 6.
    */
   def sumOfList(l: List[Int]): Int = {
-    error("fix me")
+    l.foldLeft(0)(_ + _)
```

Comment:
foldLeft was something I didn't figure out for a while, so good for you for figuring out how to use it :)